account_invoice_production_lot/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Invoice Production Lots
+===========
+
+This module prints, for each (customer) invoice line, the delivered production
```

Comment:
@pedrobaeza the purpose of this module is to print invoice report with delivered serial numbers. I can't see the use case where incoming lots could be useful